src/libp2p_stream_multistream.erl

```diff
@@ -157,9 +125,8 @@ negotiate(client, {packet, Packet}, Data = #state{}) ->
                 not_found ->
                     negotiate(client, {error, no_handlers}, Data);
                 {Key, _} ->
-                    lager:debug("Client negotiating next handler: ~p", [Key]),
+                    lager:notice("Client negotiating next handler: ~p", [Key]),
                     {keep_state, Data#state{selected_handler = NextHandler}, [
-                        {cancel_timer, handshake_timeout},
```

Comment:
Why not cancel the timer?

Reply:
hmm i *am* cancelling the timer here, so not getting what you mean?

Reply:
Wait, the GH diff shows that you _removed_ the cancel_timer action